src/main/AndroidManifest.xml

```diff
@@ -9,8 +9,6 @@
   ~ Contact support@xandar.com.au for copyright requests.
   -->
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          android:versionCode="0"
-          android:versionName="UPDATED-BY-BUILD"
```

Comment:
I need to verify that the android-maven-plugin will consume an invalid manifest. 
They may have been there as placeholders.